ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -1474,8 +1474,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
-- as: gcp-ipi-oidc-arm-f28-destructive
-  cron: 24 10 6 * *
+- as: gcp-ipi-oidc-mini-perm-rt-fips-amd-regen-cert-f28
```

Comment:
```suggestion
- as: gcp-ipi-oidc-amd-f28
  cron: 39 13 17 * *
  steps:
    cluster_profile: gcp-qe
    env: 
      EXTRACT_MANIFEST_INCLUDED: "true"
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-minimal-permission
```

Reply:
@jianlinliu I saw 4.15 / 4.17 / 4.18 are all with `gcp-ipi-oidc-mini-perm-rt-fips-amd-regen-cert-f28`, do you mean to let 4.16 alone with `gcp-ipi-oidc-mini-perm-fips-amd-f28`? 
```
$ grep -E "as: gcp-ipi-oidc" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml
- as: gcp-ipi-oidc-mini-perm-rt-fips-amd-regen-cert-f28
- as: gcp-ipi-oidc-mini-perm-arm-f28-destructive
$ grep -E "as: gcp-ipi-oidc" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml
- as: gcp-ipi-oidc-mini-perm-rt-fips-amd-regen-cert-f28
- as: gcp-ipi-oidc-mini-perm-arm-f28-destructive
$ grep -E "as: gcp-ipi-oidc" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml
- as: gcp-ipi-oidc-mini-perm-rt-fips-amd-regen-cert-f14
- as: gcp-ipi-oidc-mini-perm-arm-f28-destructive
$ grep -E "as: gcp-ipi-oidc" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml
- as: gcp-ipi-oidc-mini-perm-rt-fips-amd-regen-cert-f28
- as: gcp-ipi-oidc-mini-perm-arm-f28-destructive
$

Reply:
okay. got it.